cargo

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env bash
 
+# TODO(klykov): this file is depricated and will be deleted. It is left to split changes on the scripts and github workflow sides (see PR#27018)
```

Comment:
I don't think we should commit to deleting this file yet, it has external users.  Also this new TODO sucks.

It may be that this file turns into 
```
#!/usr/bin/env bash
echo "Notice: please run `cargo` directly. This cargo wrapper script will be removed in the future"
exec cargo "$@"
```

Reply:
Good to know, I will modify it accordingly